processing/src/test/java/org/apache/druid/query/groupby/NestedQueryPushDownTest.java

```diff
@@ -394,27 +394,26 @@ public void testSimpleDoubleAggregation()
         .builder()
         .setDataSource("blah")
         .setQuerySegmentSpec(intervalSpec)
-        .setDimensions(Lists.newArrayList(
+        .setDimensions(
             new DefaultDimensionSpec("dimA", "dimA"),
```

Comment:
Unnecessary breakdown,
```
.setDimensions(new DefaultDimensionSpec("dimA", "dimA"), new DefaultDimensionSpec("dimB", "dimB"))
```
fits 120 columns. Also in many cases below.

Reply:
Will remove the setDimensions call. Regarding 120 columns, is that something that can be updated in the coding preferences file? I imported the druid code style file in intellij and formatted the code. So looks like this preference is not there in the style file.

Reply:
120 columns is the default IntelliJ setting so that it's not present in the config (IntelliJ automatically removes settings that correspond to the default).

Reformatting may not help because those lines are already broken down, I haven't seen IntelliJ to be able to un-break expressions in fewer lines in such cases. Also I *did* see IntelliJ to break lines that fit the column limit during reformat, that is probably a bug in IntelliJ.

Reply:
Well that is unfortunate. Without code editors being able to do this automatically, adhering to this coding guideline makes it painful. I changed formatting at a few places, but considering I had to do it manually, I am sure I may have missed some. Having said that, I see tons of place in the existing code base where this coding guideline hasn't been followed (likely because Intellij doesn't do the un-breaking).

Reply:
Yes. Hopefully IntelliJ improves on the code formatting front, currently this is definitely not the strongest side of this IDE.

Also if you write the code according to the Druid style from the beginning, it's less painful, although you still have to fix something after IntelliJ.

Yes, formatting is not followed in a lot of places because it wasn't enforced until recently, and still not enforced fully because either there is no corresponding Checkstyle rule, or exactly for the reason that you noted - tons of existing violations makes it difficult to introduce the rule.